### PR TITLE
fix custom input rendering

### PR DESCRIPTION
**Pull request by BrianMarre**
fixes an bug encountered when trying to render a PICMI-simulation after adding custom input to it.

To avoid a repeat also adds a test case that test that a simulation with custom input may be rendered.

### CODE REVIEW

**Comment by PrometheusPi** [comment]
@psychocoderHPC There seems to be something wrong with the CI. It says 
> Currently unable to fetch data for this pipeline.

**Comment by BrianMarre** [comment]
The CI got confused because I tried to install the json dependency which does not exist. Json is actually already included in the core python install.

**Comment by BrianMarre** [comment]
@PrometheusPi ready for review

File: lib/python/picongpu/pypicongpu/simulation.py [hunk 1]
```diff
@@ -98,11 +99,13 @@ def __render_custom_user_input_list(self) -> dict:
...
 
     def __foundCustomInput(self, serialized: dict):
-        print("NOTE: found custom user input with tags: " + str(serialized["customuserinput"]["tags"]))
-        print(
-            "\t WARNING: custom input is not checked, it is the users responsibility to check inputs and generated input."
+        logging.info(
+            "found custom user input with tags: "
+            + str(serialized["customuserinput"]["tags"])
+            + "\n"
+            + "\t WARNING: custom input is not checked, it is the users responsibility to check inputs and generated input.\n"
```
Comment (PrometheusPi) [inline lib/python/picongpu/pypicongpu/simulation.py]: typo 
```suggestion
            + "\t WARNING: custom input is not checked, it is the user's responsibility to check inputs and generated input.\n"
```

**Review by PrometheusPi — DISMISSED** [review]

File: lib/python/picongpu/pypicongpu/simulation.py (hunk 1 above)
Comment (PrometheusPi) [reply lib/python/picongpu/pypicongpu/simulation.py]: Looks good, could just fix the typo.

**Review by PrometheusPi — APPROVED** [review]

**Pull request merged**